src/main/java/org/jboss/xavier/integrations/DecisionServerAutoConfiguration.java

```diff
@@ -41,8 +43,10 @@ public XStreamDataFormat xStreamDataFormat() {
         XStream xstream = BatchExecutionHelper.newXStreamMarshaller();
         // Use the "model" package instead of the one used on the kie server
         xstream.aliasPackage(MIGRATION_ANALYTICS_INPUT_MODELS_PACKAGE_NAME, UploadFormInputDataModel.class.getPackage().getName());
+        xstream.aliasPackage(MIGRATION_ANALYTICS_INPUT_MODELS_PACKAGE_NAME + ".workload.inventory", VMWorkloadInventoryModel.class.getPackage().getName());
         xstream.aliasPackage(MIGRATION_ANALYTICS_OUTPUT_MODELS_PACKAGE_NAME, ReportDataModel.class.getPackage().getName());
         xstream.aliasPackage(MIGRATION_ANALYTICS_OUTPUT_MODELS_PACKAGE_NAME, InitialSavingsEstimationReportModel.class.getPackage().getName());
+        xstream.aliasPackage(MIGRATION_ANALYTICS_OUTPUT_MODELS_PACKAGE_NAME , WorkloadInventoryReportModel.class.getPackage().getName());
```

Comment:
In the end, it turned out this is a mapping you can create to "translate" classes from one package to another.

In our case, [`WorkloadInventoryReportModel`](https://github.com/project-xavier/xavier-analytics/blob/master/src/main/java/org/jboss/xavier/analytics/pojo/output/workload/inventory/WorkloadInventoryReportModel.java), coming from `xavier-analytics` rules, belongs to `org.jboss.xavier.analytics.pojo.output.workload.inventory` package.

At the same time, the `WorkloadInventoryReportModel` in `xavier-integration` is in `org.jboss.xavier.analytics.pojo.output` package.

With this `aliasPackage` method package we can tell XStream that when unmarshalling objects from `org.jboss.xavier.analytics.pojo.output.workload.inventory` package the Java classes should be searched in `WorkloadInventoryReportModel.class.getPackage().getName()` (i.e. `org.jboss.xavier.analytics.pojo.output`) package and so now it works.

For sure this could be considered also a workaround because refactoring `WorkloadInventoryReportModel` from `org.jboss.xavier.analytics.pojo.output` to `org.jboss.xavier.analytics.pojo.output.workload.inventory` maybe it's a better solution.
At least, this helped in understanding how `xstream.aliasPackage` method works and when it's useful.
```suggestion
        xstream.aliasPackage("org.jboss.xavier.analytics.pojo.output.workload.inventory" , WorkloadInventoryReportModel.class.getPackage().getName());
```

Reply:
I'm more into moving the class to the right package..... are you ok ? I have pushed those changes.

Reply:
And also added your line too.